data/data/openstack/lb/main.tf

```diff
@@ -55,7 +55,7 @@ data "ignition_systemd_unit" "local_dns" {
 Description=Internal DNS server for running OpenShift on OpenStack
 
 [Service]
-ExecStart=/bin/podman run --name bootstrap-dns --rm -t -i -p 53:53/tcp -p 53:53/udp -v /etc/openshift-hosts:/etc/openshift-hosts:z --cap-add=NET_ADMIN docker.io/andyshinn/dnsmasq:latest --keep-in-foreground --log-facility=- --log-queries --no-resolv --addn-hosts=/etc/openshift-hosts --server=10.0.0.2 ${replace(join(" ", formatlist("--srv-host=_etcd-server-ssl._tcp.${var.cluster_name}.${var.cluster_domain},${var.cluster_name}-etcd-%s.${var.cluster_domain},2380,0,10", var.master_port_names)), "master-port-", "")}
+ExecStart=/bin/podman run --name bootstrap-dns --rm -t -i -p 53:53/tcp -p 53:53/udp -v /etc/openshift-hosts:/etc/openshift-hosts:z --cap-add=NET_ADMIN docker.io/andyshinn/dnsmasq:latest --keep-in-foreground --log-facility=- --log-queries --no-resolv --addn-hosts=/etc/openshift-hosts --server=10.0.0.2 ${replace(join(" ", formatlist("--srv-host=_etcd-server-ssl._tcp.${var.cluster_name}.${var.cluster_domain},${var.cluster_name}-etcd-%s.${var.cluster_domain},2380,0,10", var.controlplane_port_names)), "controlplane-port-", "")}
```

Comment:
I think this entry would benefit from line-continuation like we have [here][1].  Are there any issues with that when it's inline HCL?

[1]: https://github.com/openshift/installer/blob/76f91bdfe4e183526e19b0afc6bf85fb1f140b54/data/data/bootstrap/systemd/units/kubelet.service.template#L11-L22